tests/cleaner_tests/full_report/full_report_run.py

```diff
@@ -45,6 +45,19 @@ def pre_sos_setup(self):
             % (host.lower(), host.upper(), short.lower(), short.upper())
         )
 
+    def test_default_mapping(self):
+        self.assertFileExists('/etc/sos/cleaner/default_mapping')
+        self.assertOutputContains('Wrote mapping to')
+        with open('default_mapping') as ref:
```

Comment:
What file you refer to here? When I run this test I get:

    (1/8) tests/cleaner_tests/full_report/full_report_run.py:FullCleanTest.test_default_mapping: ERROR: [Errno 2] No such file or directory: 'default_mapping' (2.26 s)


Also I dont understand what you aim to compare here. `self.files[0][1]` should be `/etc/sos/cleaner/default_mapping` which will be populated (from scratch) by a mapping of `sos report --clean` - what content are you trying to compare with?

Maybe it makes sense to check if the file contains (some? all?) from `hostname_map` / `ip_map` / `ipv6_map` / `mac_map` / `keyword_map` / `username_map` strings (which should be in fact keys for nested maps)?

Reply:
Yea that was what I was going for but I did not consider that default_mapping would generate machine specific information. Updating commit to change that